catalog-rest-service/src/main/resources/ui/src/pages/DatasetDetailsPage/DatasetDetailsPage.component.tsx

```diff
@@ -247,15 +309,20 @@ const DatasetDetailsPage: FunctionComponent = () => {
         setIsLoading(false);
       });
 
-    setActiveTab(1);
-    getLineageByFQN(tableFQN, EntityType.TABLE).then((res: AxiosResponse) =>
-      setEntityLineage(res.data)
-    );
+    getLineageByFQN(tableFQN, EntityType.TABLE)
+      .then((res: AxiosResponse) => {
+        setEntityLineage(res.data);
+        setIsLineageLoading(false);
+      })
+      .catch(() => {
+        setIsLineageLoading(false);
```

Comment:
`setIsLineageLoading(false)` is present in `then` as well as `catch`, so it should be moved out in `finally` block

Reply:
updated